browser/js/global.js

```diff
@@ -108,7 +108,8 @@ window.app = {
 	var chromebook = window.ThisIsTheAndroidApp && window.COOLMessageHandler.isChromeOS();
 
 	var touch = !window.L_NO_TOUCH && (pointer || 'ontouchstart' in window ||
-			(window.DocumentTouch && document instanceof window.DocumentTouch)) && !chromebook;
+			(window.DocumentTouch && document instanceof window.DocumentTouch) ||
+			window.matchMedia('(pointer: coarse)').matches) && !chromebook;
```

Comment:
How do we handle chromebooks with touchscreen?

Reply:
> How do we handle chromebooks with touchscreen?

that's a good question! unfortunately I don't have a chromebook to test on, but from line 108 here
```js
var chromebook = window.ThisIsTheAndroidApp && window.COOLMessageHandler.isChromeOS();
```
chromebook detection is whether we're running on the android app on chromeos... therefore the detection in the browser would be handled exactly the same way as on a windows laptop that had a touchscreen (assuming ChromeOS doesn't appear as a touchscreen, which I guess it could)

Reply:
I'm not sure how the android app would handle input on a chromebook, I remember being told that there was something a little different for the iOS app... maybe android has something similar?